packages/vue-pdf-viewer/package.json

```diff
@@ -31,30 +31,30 @@
     "dependencies": {
         "@certifaction/pdfjs": "^2.0.1",
         "@mdi/js": "^7.2.96",
-        "core-js": "^3.30.1"
+        "core-js": "^3.31.0"
     },
     "peerDependencies": {
         "vue": "^3.3",
-        "vue-i18n": "^9.2"
+        "vue-i18n": "^9.3.0-beta.20"
```

Comment:
beta.21 was released yesterday, but as it's a peerDependency it's not that important.

Reply:
Updated